<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>business</class>
 <widget class="QWidget" name="business">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>支行名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用户数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总金额</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>业务统计</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1" colspan="2">
       <widget class="QDateEdit" name="dateEdit_2">
        <property name="date">
         <date>
          <year>2021</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>结束时间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>贷款</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>起始时间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>储蓄</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QDateEdit" name="dateEdit">
        <property name="date">
         <date>
          <year>2020</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>业务分类</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Search</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
